app/views/admin/navigation_links/_form.html.erb

```diff
@@ -24,6 +24,20 @@
     <div data-svg-icon-upload-target="svgIconMessageValidate"></div>
   </div>
 </div>
+<div class="form-group">
+  <%= form.label :section %>
+  <div class="grid gap-4 grid-cols-2 l:grid-cols-3">
+    <label class="rich-selector crayons-field crayons-field--radio">
+      <%= form.radio_button :section, "default", checked: true, class: "crayons-radio" %>
+      <div class="crayons-field__label">Default</div>
+    </label>
+    <label class="rich-selector crayons-field crayons-field--radio">
+      <%= form.radio_button :section, "other", class: "crayons-radio" %>
+      <div class="crayons-field__label">Other</div>
+    </label>
+  </div>
+  <div class="alert alert-info">Determines which section the link will be displayed in</div>
```

Comment:
suggestion (not blocking) - This is really useful context, and we could link it with the new `fieldset` proposed above to make sure screen reader users benefit from it too! We'd just need to give this div an id, and then on the fieldset add `aria-describedby="id-of-the-description"`. Adding that will mean that when screen reader users arrive at the form group, the extra hint will be read to them too.